gitreceived/gitreceived.go

```diff
@@ -197,13 +201,22 @@ func handleChannel(conn *ssh.ServerConn, newChan ssh.NewChannel) {
 				ch.Stderr().Write([]byte("Invalid repo.\n"))
 				return
 			}
-
-			if err := ensureCacheRepo(cmdargs[1]); err != nil {
+			tempDir := *repoPath
+			if *useBlobstore {
+				path, err := ioutil.TempDir("", "")
```

Comment:
I need the `tempDir` variable available later when I start uploading the new cache.